frontend/styles/var/fonts.scss

```
@@ -10,5 +10,5 @@ $large: px-to-rem(24);
 $regular: 400;
 $bold: 600;
 
-$line-height: 1.5;
+$line-height: $line-height;
```

Stylelint (17.10.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)